kafka/README.md

```diff
@@ -126,6 +126,8 @@ Also make sure the examples in README.md work.
 Convert any examples using `github.com/confluentinc/confluent-kafka-go/kafka` to use
 `gopkg.in/confluentinc/confluent-kafka-go.v1/kafka` import path.
 
+    $ find examples/ -type f -name "*\.go" -exec sed -i -e "s/github\.com\/confluentinc\/confluent-kafka-go\/kafka/gopkg\.in\/confluentinc\/confluent-kafka-go\.v1\/kafka/" {} +
```

Comment:
tip: use another regex substituion separator when working with paths, e.g::
`sed -i -e 's,github.com/confluentinc/confluent-kafka-go/....,some-replacement,g'`

micro nit: use single-quotes instead of double-quotes
nit: should replace all occurrences since we can't know if the import is the first one, so use `s,...,..,g`

Reply:
what's the trailing + for?

Reply:
`-exec cat {} +` results in `cat foo bar`, while `-exec cat {} \;` results in `cat foo; cat bar`, i.e. + uses a single sed instead of multiple. Does not matter.